relayer/provider/cosmos/provider.go

```diff
@@ -1208,6 +1226,15 @@ func acknowledgementsFromResultTx(dstChanId, dstPortId, srcChanId, srcPortId str
 	return nil, fmt.Errorf("no packet data found")
 }
 
+// splitEventKey splits the keys from a map of events where the keys are event.Type+"."+attribute.Key
+// and returns both the event.Type and attribute.Key.
+func splitEventKey(key string) (string, string) {
+	stuff := strings.Split(key, ".")
```

Comment:
I don't know what the attribute keys look like, but it seems like this should be `strings.SplitN(key, ".", 2)`.

I usually name the variable `parts` which is arguably more descriptive than `stuff` :)

Reply:
hehe can't argue with that :P 

I copied this from another spot in the relayer where events are parsed like this. The event parsing stuff could use some work especially since we will be doing more of this for Interchain Accounts and #568